pkg/cmd/openshift-tests/dev/dev.go

```diff
@@ -153,6 +152,9 @@ a running cluster.
 `),
 
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if true {
+				return fmt.Errorf("this command got nerfed")
+			}
```

Comment:
I have to at least ask, you really want this in here?

Reply:
What is this command trying to do and does it make sense this way with invariant tests?  Would a rewrite when we need it again make more sense?

I'm up for removing the command of you're down with it

Reply:
As far as removing it goes I would be interested on checking with @dgoodwin to see what his original intent was and if it is still of use. If leaving it disabled then a comment regarding why it was disabled would suffice.

Reply:
This command lets developers work on disruption test code and run it against a given intervals file containing disruption we're interested in testing against, getting feedback in seconds rather than hours waiting for CI to run. I would prefer to keep it operational.

With your new testing, would it be possible to run through the junit generation parts of your interface, but skip the setup/generate intervals portions.

Reply:
It is supposed to run the "should remain available" disruption tests against the historical data on disk. It assumes it's already given an intervals file with the observed disruption. In your list, #5, plus running the tests and viewing the output to see what would fail and with what values.

Deep is going to need similar very soon for alerts.